Gemfile

```diff
@@ -70,3 +70,7 @@ group :test do
   gem "selenium-webdriver"
   gem "webdrivers"
 end
+
+gem "webpacker"
```

Comment:
Whenever you add any new gem, lock them with a version number.
You can use the current version mentioned in gemfile.lock